charts/kubernikus-system/charts/prometheus/kubernikus.alerts

```diff
@@ -13,6 +13,18 @@ groups:
       description: Kluster {{ $labels.kluster_id }} is stuck in {{ $labels.phase }} for 1h
       summary: Kluster stuck in phase {{ $labels.phase }}
 
+  - alert: KubernikusManyKlusterStuckInPending
+    expr: round((count(kubernikus_kluster_status_phase{phase="Pending"} == 1)) / count(kubernikus_kluster_info) * 100) >= 20
```

Comment:
This would rarely fire. Assume we have 50 klusters. We would need to have 10 `pending` clusters until we see the alert.

```suggestion
 - alert: KubernikusKlusterStuckInPending
    expr: kubernikus_kluster_status_phase{phase="Pending"} == 1
    for: 168h
    labels:
      tier: kubernikus
      service: k8s
      severity: info
      context: kluster
    annotations:
      description: Kluster {{ $labels.kluster_id }} is stuck in {{ $labels.phase }} for more than a week. Check the project and delete abandoned klusters.
      summary: Kluster stuck in phase {{ $labels.phase }}
```